src/shared/components/driverAnnotations/DriverAnnotationControls.tsx

```diff
@@ -244,69 +245,74 @@ export default class DriverAnnotationControls extends React.Component<
+                                                data-test="annotateCBioPortalCount"
+                                            />
+                                            cBioPortal {'>'}=
+                                        </label>
+                                        <EditableSpan
+                                            value={
                                                 this.props.state
-                                                    .annotateDriversCBioPortal
+                                                    .annotateCBioPortalInputValue ||
+                                                ''
                                             }
-                                            onClick={this.onInputClick}
-                                            data-test="annotateCBioPortalCount"
+                                            setValue={
+                                                this.props.handlers
+                                                    .onChangeAnnotateCBioPortalInputValue
+                                            }
+                                            numericOnly={true}
+                                            textFieldAppearance={true}
+                                            maxChars={10}
                                         />
-                                        cBioPortal {'>'}=
-                                    </label>
-                                    <EditableSpan
-                                        value={
-                                            this.props.state
-                                                .annotateCBioPortalInputValue ||
-                                            ''
-                                        }
-                                        setValue={
-                                            this.props.handlers
-                                                .onChangeAnnotateCBioPortalInputValue
-                                        }
-                                        numericOnly={true}
-                                        textFieldAppearance={true}
-                                        maxChars={10}
-                                    />
-                                </div>
-                            )}
-                            {this.props.handlers
-                                .onChangeAnnotateCOSMICInputValue && (
-                                <div className="checkbox">
-                                    <label>
-                                        <input
-                                            type="checkbox"
-                                            value={EVENT_KEY.annotateCOSMIC}
-                                            checked={
+                                    </div>
+                                )}
+                            {getServerConfig().show_oncokb_cbiocosmic &&
```

Comment:
Update property names after update of backend branch.

Reply:
fixed